src/Phinx/Db/Adapter/PdoAdapter.php

```diff
@@ -393,6 +394,50 @@ public function insert(Table $table, $row)
     /**
      * {@inheritdoc}
      */
+    public function update(Table $table, $row, $whereColumns)
+    {
+        $this->startCommandTimer();
+        $this->writeCommand('update', array($table->getName()));
+
+        if (is_string($whereColumns)) {
+            $whereColumns = [$whereColumns];
+        }
+        if (!is_array($whereColumns)) {
+            throw new InvalidArgumentException;
+        }
+
+        $whereValues = [];
+        $whereParams = [];
+        foreach ($whereColumns as $column) {
+            // Store value for where clause
+            $whereValues[$column] = $row[$column];
+            // Set param name and quote column
+            $whereParams[] = $this->quoteColumnName($column) . "=?";
+            // Remove where values as they shouldn't be updated
+            unset($row[$column]);
```

Comment:
Ahhh.. I'm REALLY torn about how to handle the where clause. I initially thought it may be necessary to ONLY allow primary key values to keep things simple. But then that is next to useless for the majority of data changes where you want to filter on other columns. I think I'm running in circles on this.